scripts/issue/633/migrate_baurabal_strategy.py

```diff
@@ -0,0 +1,97 @@
+from brownie import interface, chain, accounts
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+from rich.console import Console
+
+C = Console()
+
+AURABAL_STRAT_CURRENT = registry.eth.strategies._deprecated["native.bauraBal"]
+AURABAL_STRAT_NEW = registry.eth.strategies["native.bauraBal"]
+BAURA_BAL = registry.eth.sett_vaults.bauraBal
+DEV_MULTI = registry.eth.badger_wallets.dev_multisig
+BADGERTREE = registry.eth.badger_wallets.badgertree
+TROPS = registry.eth.badger_wallets.treasury_ops_multisig
+
+USER = "0xdE0AEf70a7ae324045B7722C903aaaec2ac175F5"
+
+def main(simulation="false"):
+    safe = GreatApeSafe(registry.eth.badger_wallets.dev_multisig)
+    auraBal_strat_current = interface.IAuraBalStaker(AURABAL_STRAT_CURRENT, owner=safe.account)
+    auraBal_strat_new = interface.IAuraBalStaker(AURABAL_STRAT_NEW, owner=safe.account)
+    auraBal_vault = interface.ITheVault(BAURA_BAL, owner=safe.account)
+    want = interface.IERC20(auraBal_vault.token())
+    bb_a_usd = interface.IERC20(auraBal_strat_new.BB_A_USD())
+
+    ## Check integrity
+    assert auraBal_vault.strategy() == auraBal_strat_current.address
+
+    assert auraBal_strat_new.want() == auraBal_strat_current.want()
+    assert auraBal_strat_new.vault() == auraBal_strat_current.vault()
+    assert (
+        auraBal_strat_new.withdrawalMaxDeviationThreshold()
+        == auraBal_strat_current.withdrawalMaxDeviationThreshold()
+    )
+    assert auraBal_strat_new.autoCompoundRatio() == auraBal_strat_current.autoCompoundRatio()
+    assert (
+        auraBal_strat_new.claimRewardsOnWithdrawAll()
+        == auraBal_strat_current.claimRewardsOnWithdrawAll()
+    )
+    assert (
+        auraBal_strat_new.balEthBptToAuraBalMinOutBps()
+        == auraBal_strat_current.balEthBptToAuraBalMinOutBps()
+    )
+
+    ## Harvest current strat
+    prev_bbausd_strat_balance = bb_a_usd.balanceOf(AURABAL_STRAT_CURRENT)
+    auraBal_strat_current.harvest()
```

Comment:
im getting a revert here?

Reply:
Working for me. Can you share the trace?